src/platform/qt/SettingsView.h

```diff
@@ -80,7 +81,7 @@ private slots:
 
 	ConfigController* m_controller;
 	InputController* m_input;
-	ShaderSelector* m_shader = nullptr;
+	QPointer<ShaderSelector> m_shader = nullptr;
```

Comment:
Assuming we do use QPointer, the explicit initialization isn't needed anymore, as QPointer<T> isn't POD and the default constructor sets it to null.

Reply:
Good point.

Reply:
Fixed.